src/audio_unit/pipeline.rs

```diff
@@ -14,7 +12,7 @@ impl Pipeline {
         Pipeline::new(audio_units)
     }
 
-    fn new(audio_units: Vec<audio_unit::Boxed>) -> Result<Self> {
+    pub fn new(audio_units: Vec<audio_unit::Boxed>) -> Result<Self> {
```

Comment:
This is public now for use when making pipelines that function as individual Pedals in a larger pipeline.

Reply:
(in your comment) "Pedals" -> "Effects"? If not, what does "Pedals" mean at this point?

Reply:
Sorry, I mean "audio units" there.